xml/art_sles_rpi_quick.xml

```diff
@@ -6,25 +6,27 @@
 [
   <!ENTITY % entities SYSTEM "entity-decl.ent">
   %entities;
-  <!ENTITY productnamex "&sls;">
-  <!ENTITY productnumberx "15">
+  <!ENTITY productnamearch "<phrase xmlns='http://docbook.org/ns/docbook' role='productname'><phrase os='sles'>&sls; for &arm;</phrase></phrase>">
+  <!ENTITY appliancename "&productnamearch; for the &rpi;">
+  <!ENTITY jeos-firstboot "&jeos; First Boot Assistant">
 ]>
 <article role="sbp" xmlns="http://docbook.org/ns/docbook"
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
   xml:id="art.rpiquick" xml:lang="en">
  <info>
-  <title>&rpi; Quick Start</title><productname>&productname; &productnumber;</productname>
+  <title>&rpi; Quick Start</title>
+  <productname>&productnamearch; &productnumber;</productname>
   <author><personname><firstname>Fabian</firstname><surname>Vogt, Release Engineer, SUSE</surname></personname>
   </author>
   <author><personname><firstname>Jay</firstname><surname>Kruemcke, Product Manager, SUSE</surname></personname>
   </author>
   <author><personname><firstname>Andreas</firstname><surname>Faerber, Project Manager, SUSE</surname></personname>
-  </author><date>
-<?dbtimestamp format="B d, Y"?></date>
+  </author>
+  <date><?dbtimestamp format="B d, Y"?></date>
   <abstract>
    <para>
-    This guide contains an overview of &productname; for ARM on the &rpi;*
+    This guide contains an overview of &productnamearch; on the &rpi;*
```

Comment:
Is the asterisk after &rpi; (and the other terms like Broadcom etc.) a wildcard?

Reply:
No, it marks third-party trademarks. Didn't see the need to introduce an &rpireg; for that yet.